src/app/kubernetes/components/pipeline-status/pipeline-status.component.scss

```diff
@@ -1,382 +1,554 @@
-    animation: fadeInFailed 0s .7s linear forwards
-}
-
-.pipeline-status-bar.FAILED .pipeline-circle:after {
-    content: "\f00d"
-}
-
-.pipeline-status-bar.IN_PROGRESS .pipeline-line {
-    overflow: hidden
-}
-
-.pipeline-status-bar.IN_PROGRESS .pipeline-line:before {
-    animation: progress-rail 5s .5s linear infinite;
-    background-color: #0088ce;
-    transform: translateX(-100%);
-    width: 25%
-}
-
-.pipeline-status-bar.IN_PROGRESS .clip1:before,.pipeline-status-bar.IN_PROGRESS .clip2:before,.pipeline-status-bar.IN_PROGRESS .inner-circle-fill {
-    background-color: #0088ce
-}
-
-.pipeline-status-bar.IN_PROGRESS .pipeline-circle {
-    animation: fadeInProgress 0s .7s linear forwards
-}
-
-.pipeline-status-bar.IN_PROGRESS .pipeline-circle:after {
-    content: "\f021"
-}
-
-.pipeline-status-bar.NOT_EXECUTED .clip1:before,.pipeline-status-bar.NOT_EXECUTED .clip2:before,.pipeline-status-bar.NOT_EXECUTED .inner-circle-fill,.pipeline-status-bar.NOT_EXECUTED .pipeline-line:before {
-    background-color: #d1d1d1
-}
-
-.pipeline-status-bar.NOT_EXECUTED .pipeline-circle {
-    animation: fadeInNotExecuted 0s .7s linear forwards
-}
-
-.pipeline-status-bar.NOT_EXECUTED .pipeline-circle:after {
-    content: ""
-}
-
-.pipeline-status-bar.PAUSED_PENDING_INPUT .clip1:before,.pipeline-status-bar.PAUSED_PENDING_INPUT .clip2:before,.pipeline-status-bar.PAUSED_PENDING_INPUT .inner-circle-fill,.pipeline-status-bar.PAUSED_PENDING_INPUT .pipeline-line:before {
-    background-color: #f0ab00
-}
-
-.pipeline-status-bar.PAUSED_PENDING_INPUT .pipeline-circle:after {
-    content: "\f04c";
-    font-size: 10px
-}
-
-.pipeline-status-bar.ABORTED .clip1:before,.pipeline-status-bar.ABORTED .clip2:before,.pipeline-status-bar.ABORTED .inner-circle-fill,.pipeline-status-bar.ABORTED .pipeline-line:before {
-    background-color: #d1d1d1
-}
-
-.pipeline-status-bar.ABORTED .pipeline-circle {
-    animation: fadeInAborted 0s .7s linear forwards
-}
-
-.pipeline-status-bar.ABORTED .pipeline-circle:after {
-    content: "\f05e"
-}
+$build-pipeline-aborted-color: #d1d1d1;
+$build-pipeline-border-color: #d1d1d1;
+$build-pipeline-failed-color: $brand-danger;
+$build-pipeline-in-progress-color: #0088ce;
+$build-pipeline-new-color: #bee1f4;
+$build-pipeline-pending-color: #d1d1d1;
+$build-pipeline-success-color: #3f9c35;
+$build-pipeline-require-attention-color: $color-pf-gold;
+$circle-animation-time: 0.35s;
+$circle-diameter: 18px;
+$circle-border-width: ($line-border-width / 2);
+$circle-radius: ($circle-diameter / 2);
+$icon-animation-time: $inner-circle-animation-time;
+$inner-circle-animation-time: 0.1s;
+$inner-circle-color: #fff;
+$line-animation-time: 0.35s;
+$line-border-width: 8px;
+$line-grow-animation-time: 0.5s;
+$line-height: ($line-border-width / 2);
+$pipeline-padding: 10px;
+$progress-line: 100%;
+$progress-rail-animation-time: 5s;
+$semi-circle-animation-time: ($circle-animation-time / 2);
+
+// Animations
+$keyframes flexGrow {
```

Comment:
flex will be removed as part of a future major UI upgrade - can you use something other than flex to prevent us having to change this in the future?